src/server/app-env.ts

```diff
@@ -8,3 +8,17 @@ export const isFullMode = mode === AppMode.Full;
 export const isDatalensMode = mode === AppMode.Datalens;
 export const isChartsMode = mode === AppMode.Charts;
 export const isOpensourceInstallation = appInstallation === AppInstallation.Opensource;
+
+export const isZitadelEnabled = isTrueArg(process.env.ZITADEL);
+
+export const clientId = process.env.CLIENT_ID || '';
```

Comment:
Let's add env variables to server config

Reply:
Moved to server config